man/adoc/bpftrace.adoc

```diff
@@ -1021,8 +1021,8 @@ For string arguments use the `str()` call to retrieve the value.
 | comm
 | string[16]
 | 4.2
-| get_current_com
-| `comm` of the current task. Equal to the value in `/proc/<pid>/comm`
+| get_current_comm
+| `comm` of the current task. Equal to the value in `/proc/<tid>/comm`
```

Comment:
Why don't we just say thread name due to the confusion that pid in kernel space is a tid in user space.

Reply:
Thread name makes sense to me